symfony/framework-bundle/3.3/Makefile

```diff
@@ -6,9 +6,22 @@ cache-warmup: cache-clear
 	@test -f bin/console && bin/console cache:warmup || echo "cannot warmup the cache (needs symfony/console)"
 .PHONY: cache-warmup
 
-serve:
-	@echo -e "\033[32;49mServer listening on http://127.0.0.1:8000\033[39m"
+CONSOLE=bin/console
+$(CONSOLE):
+	@exit
+
+serve_as_sf: $(CONSOLE)
+	@$(CONSOLE)|grep server:start > /dev/null
+	@$(CONSOLE) server:start || exit 0
+
+	@echo "Quit the server with \033[32;49mbin/console server:stop.\033[39m"
+
+serve_as_php:
+	@echo "\033[32;49mServer listening on http://127.0.0.1:8000\033[39m";
 	@echo "Quit the server with CTRL-C."
-	@echo -e "Run \033[32mcomposer require symfony/web-server-bundle\033[39m for a better web server"
+	@echo "Run \033[32mcomposer require symfony/web-server-bundle\033[39m for a better web server"
 	php -S 127.0.0.1:8000 -t web
+
+serve:
+	@${MAKE} serve_as_sf || ${MAKE} serve_as_php
 .PHONY: serve
```

Comment:
For serve? Note that multiple .PHONY's is perfectly valid, and this was already used. The Makefiles are combined together so you don't have one absolute header section.